### DISCUSSION TITLE
Is it possible that duckdb will support cluster mode in different machines or containers?

### DISCUSSION THREAD

**Discussion by Shawn201107** [duckdb/duckdb#4546] [General]
DuckDB is an excellent in-process databases.

Just a whim. Is it possible that duckdb will support cluster mode in different machines or containers?

**Comment by hannes** [comment]
thanks, native distributed operations support in duckdb is unlikely to be added soon

**Reply by Shawn201107** [reply]
Got it, thanks

**Comment by Alex-Monahan** [comment]
Now that you can spin up a machine on AWS or similar with TB's of RAM, we are focused on the single node case for the most part! Do you have a use case in mind where one beefy server is not performing well?
